tests/jballerina-unit-test/src/test/resources/test-src/balo/test_balo/documentation/test_documentation.bal

```diff
@@ -4,3 +4,7 @@ import testDocOrg/test;
 function testDocDummy() returns (boolean) {
     return test:open("testt");
 }
+
+@test:Test
```

Comment:
According to the annotation definition this should have `{}` right? So this probably fails at compilation? 

Shall we assert that there are no compilation errors?

Reply:
Yes, assert the compile errors